js/src/components/account-card/index.js

```diff
@@ -23,7 +32,14 @@ export const APPEARANCE = {
 
 const appearanceDict = {
 	[ APPEARANCE.GOOGLE ]: {
-		Logo: GoogleLogo,
+		Logo: (
```

Comment:
💅 
How about rename to `logo` since it's changed to an instantiated component?

Reply:
This made me think, that serving an instance is not the best solution as if we are going to create two instances of `<AccountCard appearance="google">`, the Logo instance would not be duplicated and appear only in one. Do I get JSX right? I just got used to using `HTMLTemplateElement`s and instantiating them, and JSX is not a `<template>` ;)

Reply:
hm... :confused: it seems to eventually stamp two separate instances of `<img>`.

Reply:
Addressed in https://github.com/woocommerce/google-listings-and-ads/pull/897/commits/1ead489dd787acd65fedb8d7541af71a23d04594

Reply:
Sorry, I used an imprecise term. It should be called **react element** if follow [the definitions of React official blog](https://reactjs.org/blog/2015/12/18/react-components-elements-and-instances.html#elements-describe-the-tree). And the **instance** should be referred to the component instance in runtime and it keeps references to relevant DOM.